frontend/src/scenes/insights/EmptyStates/EmptyStates.tsx

```diff
@@ -83,13 +87,19 @@ function SamplingLink({ insightProps }: { insightProps: InsightLogicProps }): JS
     )
 }
 
-function QueryIdDisplay({ queryId }: { queryId?: string | null }): JSX.Element | null {
+function QueryIdDisplay({
+    queryId,
+    compact = false,
+}: {
+    queryId?: string | null
+    compact?: boolean
```

Comment:
@raquelmsmith I removed the `compact` option recently, I think we should not use it anymore, I'm now adding the `queryId` at the bottom of the available space at all times

Reply:
Thanks, the merge was a bit messy, appreciate the thorough review, I'll get these cleaned up.